js/components/user/delete-user-modal.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+<modal :title="_('Confirm deletion')"
+    class="modal-danger"
+    v-ref:modal>
+
+    <div class="modal-body">
+        <p class="lead text-center">
+            {{ _('You are about to delete the profile of the user {first_name} {last_name}.',
+                 {first_name: user.first_name, last_name: user.last_name}) }}
+        </p>
+        <p class="lead text-center">
+            {{ _('It will not be possible to recover this profile once deleted.') }}
+        </p>
+        <p class="lead text-center">
+            {{ _('Are you sure ?') }}
+        </p>
+    </div>
+
+    <footer class="modal-footer text-center">
+        <button type="button" class="btn btn-warning btn-flat pointer pull-left"
+            @click="confirm">
+            {{ _('Confirm') }}
+        </button>
+        <button type="button" class="btn btn-danger btn-flat pointer"
+            data-dismiss="modal">
+            {{ _('Cancel') }}
+        </button>
+    </footer>
+</modal>
+</template>
+
+<script>
+import API from 'api';
+
+export default {
+    components: {
+        modal: require('components/modal.vue')
+    },
+    data() {
+        return {
+            user: {}
+        };
+    },
+    methods: {
+        confirm: function() {
+            API.users.delete_user(
+                {user: this.user.id},
+                response => {
+                    this.$refs.modal.close();
+                    this.$dispatch('notify', {
+                        icon: 'exclamation-triangle',
+                        title: this._('The user {first_name} {last_name} has been successfully deleted',
```

Comment:
Might be shorter and easier to maintain with `fullname` instead of `first_name` and `last_name`.